emails/2024-05-17.md

```diff
@@ -1,20 +1,24 @@
 This is the weekly CEO update from [DoltHub](https://www.dolthub.com/). I'm Tim, the CEO of DoltHub. 
 
-{Joke}
+As a child of the 90s, I'm a big "alternative rock" fan, though as I've gotten older I can only handle so much of the negative energy. Pearl Jam is in LA next week and I'm going. I think we're at the Rolling Stones in the 90s stage for Pearl Jam which makes me feel real old.
 
-### 
+### The Dolt 24-Hour Pledge
 
+Some of you may know that we have pledged to fix Dolt correctness bugs in 24 hours or less. I wrote [a blog explaining how we do it](https://www.dolthub.com/blog/2024-05-15-24-hour-bug-fixes/), mostly to reiterate the pledge. We're all about earning your trust and we think fixing bugs quickly is a great way to do that.
 
+### ok Considered Harmful
 
-### 
-
+For the latest in our [Golang Blog series](https://www.dolthub.com/blog/?q=golang), [Zach](https://www.dolthub.com/team#zach) wrote about [calling variables "ok" in Golang](https://www.dolthub.com/blog/2024-05-10-ok-considered-harmful/). 
 
+Reddit was [not nice to him](https://www.reddit.com/r/golang/comments/1cp04su/ok_considered_harmful/). But, we've had a couple people come by [our Discord](https://discord.gg/gqr7K4VNKe) with more positive feedback and it was picked up by [Golang weekly](https://golangweekly.com/issues/507). So, on balance, I think it's safe to read.
 
 ### DoltLab Enterprise
 
-Why you need a remote: https://www.dolthub.com/blog/2024-04-25-why-remotes/
-DoltLab Getting Started: https://www.dolthub.com/blog/2024-05-02-doltlab-getting-started/
-DoltLab Enterprise: https://www.dolthub.com/blog/2024-05-09-doltlab-enterprise-getting-started/
+DoltLab is our self-hosted version of DoltHub. DoltLab gives you all the power of a DoltHub remote for your Dolt database in your own network.
+
+We've been doing a lot of DoltLab content recently based on customer interest in the product. First, we blogged about [why a database needs a remote like DoltLab](https://www.dolthub.com/blog/2024-04-25-why-remotes/). Then, we followed that up with a Getting started guide for both [DoltLab](https://www.dolthub.com/blog/2024-05-02-doltlab-getting-started/) and [DoltLab Enterprise](https://www.dolthub.com/blog/2024-05-09-doltlab-enterprise-getting-started/). Finally, as we used the product, we thought it needed a config file, so [we built one](https://www.dolthub.com/blog/2024-05-16-doltlab-v2.1.4-installer-update/). If you've been waiting to try DoltLab, it's ready now. Get on it.
+
+[![DoltLab](../images/doltlab.png)](https://www.doltlab.com)
```

Comment:
Might look nicer to use the doltlab image we use for social previews? It's in the google drive I shared

Reply:
In the email, the big background looks way worse. I want to keep these looking like I hand crafted them in Gmail.